packages/libs/react-ui/tsconfig.esm.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "./node_modules/@kadena-dev/heft-rig/profiles/default/tsconfig-base.json",
+  "compilerOptions": {
+    "types": ["heft-jest"],
+    "target": "ES2017",
+    "esModuleInterop": true,
+    "strict": true,
+    "skipLibCheck": true,
+    "jsx": "react",
+    "module": "ESNext",
+    "declaration": true,
+    "declarationDir": "types",
+    "sourceMap": true,
+    "outDir": "dist/esm",
+    "allowJs": true,
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
+    "forceConsistentCasingInFileNames": true
+  },
+  "include": ["src/**/*"]
+}
```

Comment:
Are all the options different from the one from `./node_modules/@kadena-dev/heft-rig/profiles/default/tsconfig-base.json`?

Reply:
updated them!